tests/vpp.cpp

```diff
@@ -29,6 +29,8 @@
 
 using namespace YamiMediaCodec;
 
+#define COLOR_BALANCE_DEFAULT -1
```

Comment:
okay， thanks @xuguangxin .
#define COLOR_BALANCE_DEFAULT COLORBALANCE_LEVEL_NONE 
according to your suggestion, I hope the above piece code can work.